sonar-db/src/test/java/org/sonar/db/user/GroupTesting.java

```diff
@@ -30,8 +29,8 @@ public static GroupDto newGroupDto() {
     GroupDto group = new GroupDto()
       .setName(randomAlphanumeric(255))
       .setDescription(randomAlphanumeric(200));
-    group.setCreatedAt(new Date(nextLong()));
-    group.setUpdatedAt(new Date(nextLong()));
+    group.setCreatedAt(new Date());
```

Comment:
XXXTesting should always generate some random data : if we need some fixed data, we should manually set them and not expect the dto generated by the XXXTesting to have some known value in fields.
So I would prefer to revert these 2 changes, and to set the new Date() in your test.